zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1738,7 +1738,9 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
     (Java system properties: **zookeeper.ssl.ciphersuites** and **zookeeper.ssl.quorum.ciphersuites**)
     **New in 3.5.5:**
     Specifies the enabled cipher suites to be used in client and quorum TLS negotiation.
-    Default: Enabled cipher suites depend on the Java runtime version being used.
+    Default: None, the JVM defaults are used (3.10.0+),
+    Enabled cipher suites are hard coded, with the ordering dependent on whether Java 8, or Java 9+ is used.
+    For Java 8 the list begins with the TLSv1.2 CBC ciphers, while for Java 9+ it begins with the TLSv1.2 CBM ciphers (3.5.5-3.9.x).
```

Comment:
Shall we list the cipher suites explicitly here which were previously in the code?
I know source control preserves it for the future, but I think it could be beneficial here as well for future doc readers.

Reply:
The format of this list doesn't really allow for long readable explanations.

We could add it in a separate section maybe ?

Reply:
Sounds good to me, thanks.

Reply:
I've added a new section, @anmolnar .